dvc/repo/pull.py

```diff
@@ -18,12 +18,24 @@ def pull(
     recursive=False,
     all_commits=False,
     run_cache=False,
+    glob=False,
 ):
     if isinstance(targets, str):
         targets = [targets]
 
+    if glob:
+        from glob import iglob
+
+        expanded_targets = [
+            exp_target
+            for target in targets
+            for exp_target in iglob(target, recursive=True)
+        ]
+    else:
+        expanded_targets = targets
```

Comment:
Seems like we already have this code in `add`, would it be possible to create a helper(e.g. in `dvc/utils/__init__.py`) to reuse?

Reply:
Btw, to support glob for push/pull/fetch and `status -c`, we could just put glob-logic int `Repo.used_cache` method, where it could then call our helper to expand the globs.

Reply:
yes, makes sense.